File: PrimeNumbers.py
#!/usr/bin/python
from math import sqrt # import squareroot module
n=int(100000) # number of primes to generate
z=n*100 # allows large enough primes for n primes
PN=[2,3,5] # array into which primes will be stored
for i in range(7,z,2): # loops through odd numbers to be check for prime
    for ii in range(0,int(sqrt(len(PN))),1): # loops through "PN" values to test i
        w = i%PN[ii] # modulus check on i with each value "PN"
        if (w==0): break # if i can be evenly divided by a value "PN", get new i
    if(w<>0):PN.append(i) # if i cannot be evenly divided by "PN", insert i into "PN"
    x=(len(PN)); # get the number of values in "PN"
    if(x==n):break # exits program once "n" primes have been generated
for z in PN:print z # print all primes to terminal

# Tested on March 1, 2018
#
# Script Data:
# Generated 100,000 prime numbers starting with 2 and ending with 1,299,709
#
# TIME: real	0m6.200s
# TIME: user	0m6.124s
# TIME: sys	0m0.052s
#
# OS: Amazon Linux
# vendor_id	: GenuineIntel
# model name	: Intel(R) Xeon(R) CPU E5-2676 v3 @ 2.40GHz
# cpu MHz		: 2394.562
# cache size	: 30720 KB
# MemTotal: 2.049 GB
#
# Python 2.7.13

File: PrimeNumbers.js
<script>
var n=100000; // check for n number of primes
var z=n*100; // allows large enough prime numbers for n primes
var PN=[2,3]; // the array into which the prime numbers will go
for (var i=3;i<=z;i=i+2) { // loops through odd numbers
  for (var ii=0; ii<Math.sqrt(PN.length); ii++) { // loops through the "PN" array
    var v=i%PN[ii]; // checks i with each value of "PN" for factorability
    if (v===0) {break;} // if i is not prime, then get the next i
  }
  if (v!==0) {PN.push(i); // if i is prime, then put it into "PN"
    console.log(i); // print the results to terminal
    if(PN.length===n){break;} // Exits once "n" primes have been generated
  }
}

// Tested on March 12, 2018
//
// Script Data:
// Generated 100,000 prime numbers starting with 5 and ending with 1,299,709
//
// TIME: 51.782 Seconds
//
// MAC OS High Sierra Version 10.13.3
// Processor 2.5 GHz Intel Core i7
// Memory 16 GB 1600 MHz DDR3
//
// Firefox Quantum 58.0.2(64 bit)
//
</script>
